includes/admin/class-wc-admin-addons.php

```diff
@@ -704,7 +704,9 @@ public static function install_woocommerce_payments_addon() {
 			'repo-slug' => 'woocommerce-payments',
 		);
 
-		WC_Install::background_installer( $services_plugin_id, $wcpay_plugin );
+		WC_Install::background_installer( $wcpay_plugin_id, $wcpay_plugin );
+
+		do_action( 'wc_addons_woocommerce_payments_installed' );
```

Comment:
Can we find out here that extension was installed from `Payments` tab of marketplace or other tabs: e.g. `Featured`, `All`, etc?

Reply:
Hey @vbelolapotkov 

This would be possible. The tab is available via the URL `section` parameter (if this doesn't exist, then the tab is `_featured`). Do you think it's worth distinguishing between the two in our event? I wasn't sure if we needed to be this granular. 

Its worth noting that the banner only appears to display in the featured and payments tabs.

Reply:
I don't have answer here, so asked in slack :)

Reply:
We got an answer in Slack and I added the section to props in https://github.com/woocommerce/woocommerce/pull/29052/commits/7a4b54ad38cad180e6353b936b080774b25a50a4